### PR TITLE
Added fancy modals

### PR DESCRIPTION
**Pull request by AMyscich**

### CODE REVIEW

**Review by TBolton2000 — APPROVED** [review]
Looks great!

File: src/client/components/ContactUs.tsx [hunk 1]
```diff
@@ -77,24 +77,25 @@ export const ContactUs: React.FC<RouteComponentProps> = () => {
 
     const onSubmit = async (formRef) => {
         // printData(firstName, lastName, phone, email, comment);
-        if (formRef.current.reportValidity()) {
-            const success = await contactUs({variables: {firstName, lastName, phoneNumber, email, comment}});
+        const success = await contactUs({variables: {firstName, lastName, phoneNumber, email, comment}});
```
Comment (TBolton2000) [inline src/client/components/ContactUs.tsx]: Oh wait we should probably check that the email is valid before making this function call

**Review by TBolton2000 — CHANGES_REQUESTED** [review]
check that the email is valid before sending the contactUs call to the backend

File: src/client/components/ContactUs.tsx (hunk 1 above)
Comment (AMyscich) [reply src/client/components/ContactUs.tsx]: I did, it should be below the highlighted lines

**Review by TBolton2000 — APPROVED** [review]
I am going to open an issue here, but this is fine for the time being

**Pull request merged**